test/functional/ui/highlight_spec.lua

```diff
@@ -540,6 +540,27 @@ describe("'listchars' highlight", function()
     ]])
   end)
 
+  it("'cursorline' and 'standout'", function()
```

Comment:
This isn't releveant to the `describe("'listchars' highlight",` group, so I moved it.